src/main/java/pl/cyber/trainees/pracaDomowa29Tydzien/ImieIWiek.java

```diff
@@ -0,0 +1,16 @@
+package pl.cyber.trainees.pracaDomowa29Tydzien;
+
+public class ImieIWiek {
+   //1. Napisz metodę, która zwróci Twój aktualny wiek.
+    public Integer wiek;
+    //2. Napisz metodę, która zwróci Twoje imię,
+    public void wczytajImie(String imie) {
+        System.out.println("Twoje imie to "+imie);
+    }
+
+    public Integer ImieIWiek(final Integer wiek) {
```

Comment:
Jeśli chciała Pani zrobić w tym miejscu konstruktor to jest on nie do końca poprawny
Oczywiście konstruktor musi mieć modyfikator dostępu jako public nazwę taką samą jak nazwa klasy i parametry takie jak klasa.
Natomiast konstruktory nie posiadają zwracanego typu

Reply:
o dziękuję !